Support for React 17

Thanks for continue maintain the Github login project 👍 😎

I have created a new react project that has the version `17.0.2`. 

If I run install command, I get an error of depending react version.
```bash
$ npm install react-login-github --save

npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: react-material-dashboard@1.0.0
npm ERR! Found: react@17.0.2
npm ERR! node_modules/react
npm ERR!   react@"^17.0.2" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.0.0-0" from react-login-github@1.0.7
npm ERR! node_modules/react-login-github
npm ERR!   react-login-github@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
```

Can we update this library to make them run on react >= 17 too? 

Thanks!

--- a/package.json
+++ b/package.json
@@ -25,7 +25,7 @@
   "license": "MIT",
   "peerDependencies": {
     "prop-types": "^15.7.2",
-    "react": "^17.0.0-0"
+    "react": "^16.0.0 || ^17.0.0 || ^18.0.0"
   },
   "devDependencies": {
     "@babel/core": "^7.9.6",

--- a/package.json
+++ b/package.json
@@ -25,7 +25,7 @@
   "license": "MIT",
   "peerDependencies": {
     "prop-types": "^15.7.2",
-    "react": "^16.0.0-0"
+    "react": "^17.0.0-0"
   },
   "devDependencies": {
     "@babel/core": "^7.9.6",

